engine/verification/assigner/engine.go

```diff
@@ -139,23 +138,48 @@ func (e *Engine) processChunk(chunk *flow.Chunk, resultID flow.Identifier) {
 	// pushes chunk locator to the chunks queue
 	ok, err := e.chunksQueue.StoreChunkLocator(locator)
 	if err != nil {
-		log.Fatal().Err(err).Msg("could not push chunk locator to chunks queue")
-		return
+		return fmt.Errorf("could not push chunk locator to chunks queue: %w", err)
 	}
 	if !ok {
 		log.Debug().Msg("could not push duplicate chunk locator to chunks queue")
-		return
+		return nil
```

Comment:
Due to duplicated chunk locator, I would suggest to return `(added bool, err error)`, so that the caller could know if there is new chunk stored, and we could log it.

Reply:
added in https://github.com/onflow/flow-go/pull/466/commits/d58c52d2c06b946b8ed9ef00a5cfd55ca0aaa145